oslc4j-client/src/main/java/org/eclipse/lyo/oslc4j/client/resources/OslcQueryParameters.java

```diff
@@ -93,7 +93,7 @@ public String getWhere() {
 	 * @param where
 	 */
 	public void setWhere(String where) {
-		this.where = encodeQueryParams(where);
+		this.where = where;
```

Comment:
Why is this a safe fix? Is there a way to unit test this? What component does the encoding now?

Reply:
I tried to find who calls these Setter methods, and I can't find any, except in the tests, See class OslcQueryResultTest.java

Otherwise, OslcQuery uses the OslcQueryParameters class, but that class doesn't call the Setters. It calls the constructor - which doesn't do any encoding.
https://github.com/eclipse/lyo.client/blob/master/oslc4j-client/src/main/java/org/eclipse/lyo/oslc4j/client/resources/OslcQuery.java

In the end, the method applyOslcQueryParams() is what sets the URI. And there you see that one uses `"result.queryParam()"`, which I trust does the necessary encoding.
https://github.com/eclipse/lyo.client/blob/705647ed999dd917a39973e20d9ed5e302dc10dd/oslc4j-client/src/main/java/org/eclipse/lyo/oslc4j/client/resources/OslcQuery.java#L156